CMakeLists.txt

```diff
@@ -15,17 +15,17 @@ set(REQ_CATKIN_PACKAGES
 find_package(catkin REQUIRED COMPONENTS ${REQ_CATKIN_PACKAGES} message_generation)
 find_package(cmake_modules REQUIRED)
 find_package(Eigen3 REQUIRED)
+set(PYBIND11_CPP_STANDARD -std=c++11)
+set(PYBIND11_PYTHON_VERSION 2.7)
+find_package(pybind11 REQUIRED)
```

Comment:
I'm concerned about making this a requirement. Perhaps make this a "if found, then build the python library" thing.

Reply:
This still fails, even with the option as CMake still looks for pybind11. I'd recommend the "if found then compile" version rather than the option version. If you want the option version, then all of the pybind11 stuff should go inside the `if` on line 189

```
CMake Error at /home/armlab/catkin_ws_test/src/sdf_tools/CMakeLists.txt:22 (find_package):
  By not providing "Findpybind11.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "pybind11",
  but CMake did not find one.

  Could not find a package configuration file provided by "pybind11" with any
  of the following names:

    pybind11Config.cmake
    pybind11-config.cmake

  Add the installation prefix of "pybind11" to CMAKE_PREFIX_PATH or set
  "pybind11_DIR" to a directory containing one of the above files.  If
  "pybind11" provides a separate development package or SDK, be sure it has
  been installed.
```